modules/ROOT/pages/configure-host.adoc

```diff
@@ -4,22 +4,33 @@ include::_attributes.adoc[]
 endif::[]
 
 Your organization is considered the host partner in Anypoint Partner Manager.
```

Comment:
When you use Anypoint Partner Manager (Partner Manager), your organization is considered the host. Configuring the host involves creating a host page (named after your organization) within which you specify the following: